Gemfile.lock

```diff
@@ -106,7 +174,40 @@ GEM
     rb-fsevent (0.11.0)
```

Comment:
I would really like to know more about fsevents if you wonderful back-enders know something about this.

Reply:
The Gemfile.lock is kind of a magical document that pulls in all the dependencies for the gems declared in the Gemfile. It also makes sure all the same gem versions are utilized by each user when they pull the repo to their remote (and run bundle install). I am do not know much about rb-fsevent